src/controlCenter/widgets/buttons/buttons.vala

```diff
@@ -0,0 +1,38 @@
+using GLib;
+
+namespace SwayNotificationCenter.Widgets {
+
+    public class Buttons : BaseWidget {
+        public override string widget_name {
+            get {
+                return "buttons";
+            }
+        }
+
+        Action[] actions;
+
+        public Buttons (string suffix, SwayncDaemon swaync_daemon, NotiDaemon noti_daemon) {
+            base (suffix, swaync_daemon, noti_daemon);
+
+            Json.Object ? config = get_config (this);
+            if (config != null) {
+                Json.Array a = get_prop_array (config, "actions");
+                if (a != null) actions = parse_actions (a);
+            }
+
+            Gtk.FlowBox container = new Gtk.FlowBox ();
```

Comment:
Set the FlowBox selection mode to none. Fixes the blue FlowBoxChild focus coloring:

![image](https://user-images.githubusercontent.com/35975961/217660962-c474293d-63d6-43ef-b55f-dee1af49c9e1.png)